### PR TITLE
Performance documentation refactoring

### PR DESCRIPTION
**Pull request by tiagojsag**

### CODE REVIEW

**Comment by tiagojsag** [comment]
@bcremer can you review this, when you have a free moment please?

File: source/developers-guide/shopware-5-performance-for-devs/index.md [hunk 1]
```diff
@@ -1,13 +1,21 @@
...
+github_link: developers-guide/shopware-5-performance-for-devs/index.md
 indexed: true
+tags:
+  - performance
+  - tips
+  - cache
 ---
 
-## Introduction
+# Shopware 5 performance guide for developers
```
Comment (bcremer) [inline source/developers-guide/shopware-5-performance-for-devs/index.md]: Header of first level is inserted by the template from the title.

File: source/sysadmins-guide/shopware-5-performance-for-sysadmins/index.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+- `MaxRequestWorkers` and `ServerLimit`: The maximum number of active server processes that are allowed to simultaneously exist on your server. Setting these values too low may cause simultaneous requests to be queued or ignored, decreasing response time during times of heavy load. Setting them too high may exhaust your server memory, causing memory swaps and decreasing performance or even crashing the server altogether. These two values should be equal. 
+
+- `MaxConnectionsPerChild`: This value represents the number of connections each thread will accept and handle before its terminated by Apache. This setting is mostly used to prevent memory leakage from consuming a significant amount of server memory. The default 0 prevents processes from ever being terminated (unless they are idle and `MaxSpareServers` has been reached, or Apache itself is terminated). If you suspect your server is affected by memory leakage, set a different value to this variable.
+
+### Nginx
```
Comment (bcremer) [inline source/sysadmins-guide/shopware-5-performance-for-sysadmins/index.md]: Please link to https://github.com/bcremer/shopware-with-nginx somewhere.

File: source/developers-guide/index.html [hunk 3]
```diff
@@ -12,7 +12,7 @@
     <li><a href="{{ site.url }}/developers-guide/shopware-5-search-bundle/">Shopware 5 SearchBundle - Easy product list selection</a></li>
     <li><a href="{{ site.url }}/developers-guide/shopware-5-core-service-extensions/">Shopware 5 core service extensions</a></li>
     <li><a href="{{ site.url }}/developers-guide/vagrant-phpstorm/">Vagrant and PHPStorm</a></li>
-    <li><a href="{{ site.url }}/developers-guide/shopware-5-performance/">Shopware 5 performance</a></li>
+    <li><a href="{{ site.url }}/developers-guide/shopware-5-performance-for-devs/">Shopware 5 performance guide for developers</a></li>
```
Comment (bcremer) [inline source/developers-guide/index.html]: Don't break existing URLs please. When needed include https://github.com/mavimo/sculpin-redirect-bundle and configure accordingly.

**Pull request merged**